modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -412,10 +412,30 @@ label:enterprise-only[]
 Incremental import is not supported for `block` format yet.
 ====
 
-When the initial data load cannot be completed in a single full import, incremental import allows the operation to be completed as a series of smaller imports.
+Incremental import allows you to incorporate large amounts of data in batches into the graph.
+You can run this operation as part of the initial data load when it cannot be completed in a single full import.
+Besides, you can update your graph by importing data incrementally, which is more performant than transactional insertion of such data.
 
-Incremental import requires the use of `--force` because it must only be run on databases in precisely the expected state and as part of an initial load.
-When an incremental import fails, it can leave the data corrupted.
+Incremental import requires the use of `--force` and can be run on an existing database only.
+
+You must stop your database, if you want to perform the incremental import within one command.
+
+If you cannot afford a full downtime of your database, split the operation into several stages:
+
+* _prepare_ stage
+* _build_ stage
+* _merge_ stage
+
+The database must be stopped for the `prepare` and `merge` stages.
+During the `build` stage the database must not be written to.
+If it is started, you have to put it into read-only mode.
+For more details, read <<import-tool-incremental-examples, Examples>>.
+
+Note that your database should have correct uniqueness constraints for the affected nodes, but the tool will tell you and exit, if such is missing.
```

Comment:
We have a similar warning in the `=== Usage and limitations`  section, which is not tested against. So, I think we could update that by adding something like: "Otherwise, the command throws an error." and remove this one here.

Reply:
To be honest, I'd prefer to keep this sentence here, in the introductory section. The idea was to explain what we mean by words 'database has to be in precisely the **expected state**'. I'd like to collect all warnings in one place, as users often don't scroll down (unfortunately). 
I agree that there is a similar warning in the section `Usage and limitations` but it's more detailed there with a cross-reference for further information.